TicTacToe/test/tictactoe/TicTacToeTest.java

```diff
@@ -3,34 +3,142 @@
 import org.junit.Test;
 import static org.junit.Assert.*;
 
+/**
+ * 
+ * @author Evan
+ */
 public class TicTacToeTest {
 
+    TicTacToe ttt;
+    /**
+     * Test of IsWinner method, of class TicTacToe.
+     */
     @Test
-    public void testIsWinner() {
+    public void testIsWinner()
+    {
+        ttt = new TicTacToe(); //Horizontal
+        for (int i = 0; i < 3; i++)
+        {
+            ttt.playMove('X', 0, i);
+        }
+        assertTrue(ttt.isWinner('X'));
+        
+        ttt = new TicTacToe(); //Vertical
+        for (int i = 0; i < 3; i++)
+        {
+            ttt.playMove('X', i, 0);
+        }
+        assertTrue(ttt.isWinner('X'));
+        
+        ttt = new TicTacToe(); //Diagonal 1
+        for (int i = 0; i < 3; i++)
+        {
+            ttt.playMove('X', i, i);
+        }
+        assertTrue(ttt.isWinner('X'));
+        ttt = new TicTacToe(); //Diagonal 2
+        for (int i = 0; i < 3; i++)
+        {
+            ttt.playMove('X', i, 2-i);
+        }
+        assertTrue(ttt.isWinner('X'));
     }
 
+    /**
+     * Test of IsFull method, of class TicTacToe.
+     */
     @Test
-    public void testIsFull() {
+    public void testIsFull()
+    {
+        ttt = new TicTacToe();
+        assertFalse(ttt.isFull());
+        for (int r = 0; r < 3; r++) //Fills the board
+        {
+            for (int c = 0; c < 3; c++)
+            {
+                ttt.playMove('X', r, c);
```

Comment:
Should also be checking that isFull == false after each move (until board is actually full)